sig/datadog/core/telemetry/emitter.rbs

```diff
@@ -7,7 +7,7 @@ module Datadog
         attr_reader http_transport: untyped
 
         extend Core::Utils::Forking
-        def initialize: (?http_transport: untyped) -> void
+        def initialize: (http_transport: untyped) -> void
```

Comment:
Minor: I think this could be pointed at `Datadog::Core::Telemetry::Http::Transport`?